test_environments/message_and_event_brokers/kafka/sasl_scram/with_tls/README.md

```diff
@@ -0,0 +1,99 @@
+# Overview
```

Comment:
```suggestion
# Testing Kafka broker with SASL_SCRAM and TLS

```

Reply:
I added `# Setup Kafka broker with SASL SCRAM authentication and TLS encryption`